### DISCUSSION TITLE
Double key pressing events configuration?

### DISCUSSION THREAD

**Discussion by offray** [sezanzeb/input-remapper#398] [Q&A]
First, thanks for input remapper. It has been pretty refreshing seeing a powerful and easy to use software for keyboard/mouse events configuration. I'm using right now for setting the middle button of my ThinkPad mouse (marked with 2 in the image below) as the Control key and it has bring a lot of ergonomy to my keyboard utilization. It is like having a cheap thumb cluster of more ergonomic keyboards.

![](https://i.imgur.com/LwgUIXd.png)

I would like to now if pressing the Input Remapper is the app to set up an extra behavior: I woulk like that pressing the ThinkPad middle key twice, makes it act as another shortcut. I'm using also sxhkd and I could try some keyboard combinations over there, but maybe some more knowledgeable users could share similar experiences on this regard.

Thanks again,

**Comment by sezanzeb** [comment]
Maybe this helps:

```
if_tap(
	if_tap(
		key(a),
		key(c)
	),
	key(b)
)
```

- press and hold: b
- press and release: c
- press twice: a

I just added this to the example docs (and updated them, they were very outdated) https://github.com/sezanzeb/input-remapper/blob/main/readme/examples.md#double-tap

**Comment by 7sparrow** [comment]
This isnt configurable in the UI is it?
